src/fluid/fluid.hpp

```diff
@@ -569,14 +569,23 @@ Fluid<Phys>::Fluid(Grid &grid, Input &input, DataBlock *datain, int n) {
       case BX1:
         VcName.push_back("BX1");
         data->vtk->RegisterVariable(Vc, outputPrefix+"BX1", BX1);
+        #if DIMENSIONS == 1
+          data->dump->RegisterVariable(Vc, outputPrefix+"Vc-BX1", BX1, IDIR);
+        #endif
         break;
       case BX2:
         VcName.push_back("BX2");
         data->vtk->RegisterVariable(Vc, outputPrefix+"BX2", BX2);
+        #if DIMENSIONS < 2
```

Comment:
while we're at it, maybe we should keep these directives consistent in style so it's easier to read ? (Isn't `DIMENSIONS < 2` exactly the same as `DIMENSIONS == 1ˋ ?)

Reply:
actually, the first DIMENSIONS == 1 should read DIMENSIONS < 1 to be coherent, and this is never true, so I cleaned it.